_includes/head.html

```diff
@@ -31,9 +31,6 @@
 
 <!--[if lt IE 9]>
```

Comment:
Yeah, it doesn't seem to work if it's not in the head. The entire page layout gets screwy.